### ISSUE TITLE
How can we load a pre-installed app?

### ISSUE THREAD

**Issue by rahuxx** [Simbul/baker#395]
I have a game/app installed on iPad and i want to load it from a screen shot inside HTMl page / book, Is it possible?

**Comment by folletto** [comment]
It's possible in the game/app you have already on the iPad implements an external protocol: you wrap the screenshot inside an `<a>` tag and you point the href to the external protocol of the other app.

For reference see:
http://stackoverflow.com/questions/419119/launch-an-app-from-within-another-iphone

Plus, the google query you can use is: https://www.google.com/search?q=ios%20call%20another%20app

**Comment by rahuxx** [comment]
HI Folletto,

Unfortuntely I am not so good in HTML 5 so can you provide any exmplae with proper code on this.

thanks in advance
rahuxx

**Comment by folletto** [comment]
Did you check the links above? :)
I don't know what I could add more... :)

**Comment by rahuxx** [comment]
I read it but I am in need to know HTML part for calling installed app. For example my app custom URL is a biology102454. now what code I should use to call this app using HTML?

I need a code in HTML5, not in xCode.

**Comment by folletto** [comment]
Ok, now I understand your doubt. :)

As I said above, it's as simple as wrapping the image in a `<a>` tag and using the protocol in its href. So for example:

`<a href="biology102454://some/path"><img src="pageview-1.jpg" /></a>`

**Comment by rahuxx** [comment]
I will try and update, thanks for your help sir folletto :)

**Comment by folletto** [comment]
You're welcome. :)

**Comment by rahuxx** [comment]
```
<head>
    <title>What Is Your Take ?</title>
    <meta name="viewport" content="width=768; initial-scale=1.0;"/>
    <link rel="stylesheet" type="text/css" href="css/book.css">
        <script language="javascript">
            function callUnity()
            {
                alert("call unity simulation");
                window.location="b-1001372-100-151-611://";
            }
        </script>
</head>
<body>
    <div id="page" class="chapter"> 
        <a class="about" href="javascript:callUnity();" target="_new">
            <img src="gfx/baker-framework-badge.png" alt="Built with Baker Ebook Framework" />
        </a>
    </div>
</body>
```

While i am using this html code,alert is running properly after that  getting below error when loading app:

2012-04-24 12:42:02.593 Baker[333:307]     Link doesn't contain param "referrer=Safari" --> open link in page
2012-04-24 12:42:02.599 Baker[333:307] • CurrPage failed to load content with error: Error Domain=WebKitErrorDomain Code=101 "The URL can’t be shown" UserInfo=0x1a8710 {NSErrorFailingURLKey=b-1001372-100-151-611://, NSErrorFailingURLStringKey=b-1001372-100-151-611://, NSLocalizedDescription=The URL can’t be shown}

app is already installed on my ipad and also i put the same ipa in the same folder where html file is.
Thanks in advanced.

**Comment by rahuxx** [comment]
no solution?

**Comment by folletto** [comment]
I probably answered too quickly they other time. We need to dig into this. :)
I reopened the issue.

Did you try to make it without the JavaScript - direct href call?

**Comment by rahuxx** [comment]
Yes i tried it without JavaScript as well, but the same error is displaying.
In above solution : `<a href="biology102454://some/path"><img src="pageview-1.jpg" /></a>`
I didn't get what you mean by "some/path" ? what should i write in place of some/path?

Thanks

**Comment by folletto** [comment]
It depends on the application: most of them use that part to send data, because the objective isn't usually just to open the application, but to open at a specific point, so most apps have a "path" to be specified.

**Comment by rahuxx** [comment]
But Sir In my Case i just want to open my app.
If you can try and provide some solution it will be great help for a newbie.

Thanks

**Comment by folletto** [comment]
Yes, so if your app doesn't require any path, you don't have to put any path. As simple as that. 

We'll check. :)

**Comment by rahuxx** [comment]
I will wait for the solution sir. Thanks for support.

**Comment by rahuxx** [comment]
any one tried it?

**Comment by rahuxx** [comment]
Hey sir folletto, did you get chance to check it?

**Comment by rahuxx** [comment]
Any update, or it is not possible?

**Comment by folletto** [comment]
I'm sorry, but we didn't have time yet to check it. Unfortunately it's an opensource project and we don't have any dedicated developer, people support this in their spare time.

**Comment by rahuxx** [comment]
VERY UNFORTUNATE :-(

**Comment by Xm4s** [comment]
Hi!

I've looked into the problem: seems that custom url schemes weren't handled correctly.
I've fixed it with commit 73032c2ac57a2550a8adb1550377736fe5e822fb, now all should work just fine :)

**Comment by rahuxx** [comment]
I Confirm !!! It Works...
Thanks xm4s and folletto.
